cmd/cdi-importer/importer.go

```diff
@@ -98,6 +101,9 @@ func main() {
 		err = importer.CopyData(dso)
 		if err != nil {
 			glog.Errorf("%+v", err)
+			if dso.RequiresScratch {
```

Comment:
Maybe have `CopyData` somehow return whether scratch is required?  Maybe it return `(bool, error)` or `err` is some sort of specific error type?

Reply:
So I didn't want to get into parsing error messages, but I can just make a type and have it implement the error interface and then check the type instead of this clumsy setup, I will change.

Reply:
Was thinking something like this:

```golang
var RequiresScratchSpaceError = fmt.Errorf("need scratch space")

// CopyData may return RequiresScratchSpaceError

// then check

if err == RequiresScratchSpaceError {

}

```

But if you want to to include more data in the error obviously a struct is the way to go

Reply:
Well not right now, but if we want more info in the future we can always switch to a struct then.